torch/onnx/symbolic.py

```diff
@@ -543,11 +545,43 @@ def reform_weights(g, w, n, intervals):
     return g.op('Concat', *slices, axis_i=0)
 
 
+def Elman_RNN_symbolic_builder(nonlinearity, input_size, hidden_size, num_layers, batch_first, dropout, bidirectional, **kwargs):
```

Comment:
I understand that `kwargs` shows up in the other RNN implementations, but is there a reason it is done this way? It seems safer to explicitly list the arguments, so that if someone adds a new argument they will notice the symbolic needs to be updated.